static/app/views/starfish/views/spans/spanSummaryPage/durationChart/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {useTheme} from '@emotion/react';
+import moment from 'moment';
+
+import usePageFilters from 'sentry/utils/usePageFilters';
+import Chart from 'sentry/views/starfish/components/chart';
+import {PERIOD_REGEX} from 'sentry/views/starfish/utils/dates';
+import {
+  useQueryGetSpanSeriesData,
+  useQuerySpansInTransaction,
+} from 'sentry/views/starfish/views/spanSummary/queries';
+import {queryDataToChartData} from 'sentry/views/starfish/views/spanSummary/sidebar';
+
+type Props = {
+  groupId: string;
+  transactionName: string;
+  spanDescription?: string;
+};
+
+function DurationChart({groupId, transactionName, spanDescription}: Props) {
+  const theme = useTheme();
+  const pageFilter = usePageFilters();
+  const {isLoading, data} = useQuerySpansInTransaction({groupId});
+
+  const spanGroupOperation = data?.[0]?.span_operation;
+  const module = data?.[0]?.module;
+  const {startTime, endTime} = getStartAndEndTime(pageFilter);
+
+  const {isLoading: isLoadingSeriesData, data: seriesData} = useQueryGetSpanSeriesData({
+    groupId,
+    spanGroupOperation,
+    transactionName,
+    description: spanDescription,
+    module,
+  });
+
+  const {p50: p50Series} = queryDataToChartData(seriesData, startTime, endTime, {
```

Comment:
I know it's still undecided, but can we start off with both p50 and p95 series on this chart to see how it looks?

Reply:
Yeah, I like that idea!